superdesk/metadata/item.py

```diff
@@ -308,6 +308,16 @@
         'type': 'string'
     },
 
+    # Desk and Stage Details
```

Comment:
I think that field will be actively used in "archive" collection whereas this field will be available in ingest. But what I can do is to move the 'task' field from archive common to metadata/item. But with this shape overwrite is fine.